lldb/test/Shell/SwiftREPL/DiagnosticOptions.test

```diff
@@ -10,8 +12,13 @@
 
 
 _ = "An unterminated string
+<<<<<<< HEAD
```

Comment:
Will fix: My mistake for not checking the git diff when I split this out of a PR from llvm/llvm-project